query-coordinator-http/src/test/scala/com/socrata/querycoordinator/TestQueryRewriterDateTruncBase.scala

```diff
@@ -1,25 +1,31 @@
 package com.socrata.querycoordinator
 
+import com.socrata.soql.SoQLAnalysis
 import com.socrata.soql.environment.ColumnName
+import com.socrata.soql.types.SoQLAnalysisType
 
 class TestQueryRewriterDateTruncBase extends TestQueryRewriterBase {
   /** Each rollup here is defined by:
     * - a name
-    * - a soql statement.  Note this must be the mapped statement, ie. non-system columns prefixed by an _, and backtick escaped
+    * - a soql statement.  Note this must be the mapped statement,
+    *   i.e. non-system columns prefixed by an _, and backtick escaped
     * - a Seq of the soql types for each column in the rollup selection
     */
   val rollups = Seq(
-    ("r_ymd", "SELECT date_trunc_ymd(`_crim-date`), `:wido-ward`, count(*) GROUP BY date_trunc_ymd(`_crim-date`), `:wido-ward`"),
-    ("r_ym", "SELECT date_trunc_ym(`_crim-date`), `:wido-ward`, count(*) GROUP BY date_trunc_ym(`_crim-date`), `:wido-ward`"),
-    ("r_y", "SELECT date_trunc_y(`_crim-date`), `:wido-ward`, count(*) GROUP BY date_trunc_y(`_crim-date`), `:wido-ward`")
+    ("r_ymd", "SELECT date_trunc_ymd(`_crim-date`), `:wido-ward`, count(*) " +
```

Comment:
Breaking this all up just to satisfy some line length constraint makes these tests a lot harder to write and work with and compare with each other.  Every change requires messing around with formatting and multiple lines... is there really benefit from breaking these up to meet some arbitrary rule?